internal/alloydbutil/engine.go

```diff
@@ -131,3 +137,78 @@ func getServiceAccountEmail(ctx context.Context) (string, error) {
 	}
 	return userInfo.Email, nil
 }
+
+// initVectorstoreTable creates a table for saving of vectors to be used with PostgresVectorStore.
+func (p *PostgresEngine) initVectorstoreTable(ctx context.Context, tableName string, vectorSize int, schemaName string, contentColumn string,
+	embeddingColumn string, metadataColumns []Column, metadataJsonColumn string, idColumn interface{}, overwriteExisting bool, storeMetadata bool) error {
```

Comment:
should idColumn really be `interface{}`?

Reply:
changed to use Column type as we discussed!